### DISCUSSION TITLE
Streetmerchant hanging in docker after Puppeteer Config

### DISCUSSION THREAD

**Discussion by JohnDoe-dev-spec** [jef/streetmerchant#1820] [Help]
Hi,

I've managed to get Streetmerchant deployed as a container in Docker. However when i change nothing in the dotenv file or environment variables the Merchant runs fine and goes through the default stores, series etc.

However, once i start changing the config/variables to what i require it then just pauses at startup after the puppeteer config. I am not sure why and hoping someone can help!

Here is an extract;

```
,> streetmerchant@ start:production /app
,> node build/src/index.js
,
,ＳＴＲＥＥＴＭＥＲＣＨＡＮＴ
,3.4.1
,
,[10:47:38 PM] info :: ℹ selected stores: amazon-it
,[10:47:38 PM] info :: ℹ selected series: 5900
,[10:47:40 PM] info :: ℹ puppeteer config:  {
,  "0": "--disable-dev-shm-usage",
,  "1": "--no-sandbox",
,  "2": "--disable-setuid-sandbox",
,  "3": "--headless",
,  "4": "--disable-gpu"
,}
```

then nothing after this, just stays here.

Thanks!

**Comment by wofnull** [comment]
could you please give some informations?
On what hardware / OS is streetmerchant showing this behaviour?
Is the needed NodeJS version installed?

**Reply by JohnDoe-dev-spec** [reply]
Hi Wofnull,

Thanks for your reply. 

I have DietPi latest version installed (Debian based OS) with latest Docker version installed along with portainer.

I pulled the nightly streetmerchant image and deployed to a container in docker. 

I assume nodeJS is installed due to me using the Streetmerchant Docker image?

Thanks

**Reply by JohnDoe-dev-spec** [reply]
Hi Wofnull,

I've fixed the issue, it was a stupid mistake on my part. My Dotenv/variable for series was set as 5900 instead of Ryzen5900, hence it hanging.

Once changed it started working like normal. Stupid me.
